### ISSUE TITLE
Momento de Sabedoria #10

### ISSUE THREAD

**Issue by vish-blog** [vish-blog/comments-new#283]
### Post de Referência:

https://www.vish.blog.br/imagens/momento-de-sabedoria-10/

**Comment by one-eyed-willy** [comment]
Concordo com o Mr. Bean, contanto que a linha entre humor e *bullying* não seja cruzada
